rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -233,6 +237,11 @@ type Payout: <"A class to represent the set of future cashflow methodologies in
     cashflow Cashflow (0..*) <"A cashflow between the parties to the trade. For interest rate and equity products, this corresponds to the FpML additionalPayment element. For credit default swaps, this corresponds to the FpML initialPayment element and the singlePayment element of the fee leg. For option products, it represents the FpML premium element.">
     performancePayout PerformancePayout (0..*) <"The performance payout, which encompasses the equity price returns, dividend returns, volatility return, variance return and correlation provisions.">
     assetPayout AssetPayout (0..*) <"The security payout when the product involves some form of securities, such as collateral in a securities financing transaction">
+    settlementCommitment SettlementCommitment (0..1)
```

Comment:
Would be good to add a comment to settlementCommitment too, like the other attributes

Reply:
updated